cli/Cargo.toml

```diff
@@ -11,3 +11,4 @@ path = "src/main.rs"
 chrono = { version = "0.4" }
 clap = "2.31"
 stellar-client = { path = "../client" }
+textwrap = "0.10"
```

Comment:
It seems like we are pulling this in to use this method https://github.com/mgeisler/textwrap/blob/master/src/lib.rs#L870-L880 . 

Kevin and I were just talking about indentation so it put a big smile on my face when I saw this, but maybe we can just pull in this one method.  Are there other things in the library we also may want to use?